### ISSUE TITLE
Needs build instructions

### ISSUE THREAD

**Issue by Gadgetoid** [rpi-ws281x/rpi-ws281x-python#2]
Title says it all. Document git submodule installation and build/install process for the library.

Note: It's unnecessary to install/use SWIG since the bindings should be regenerated any time the git submodule is updated.

**Comment by davidbradway** [comment]
@Gadgetoid, if I install via pip (`pip install rpi_ws281x`), do I need to build the submodule with `scons`?

**Comment by Gadgetoid** [comment]
@davidbradway nope, you don't need to build anything, or even touch this repository. "pip install" should grab the appropriate .whl file for your Python version - currently I'm shipping 2.7, 3.4 and 3.5 - and that contains everything you need pre-compiled and ready to go.

Build instructions are basically just for developers.

**Comment by charliesneath** [comment]
Since it’s necessary to use sudo to use the library, is it also necessary to use `sudo pip install rpi_ws281x` when installing?
